spyql/prof.py

```diff
@@ -1,9 +1,8 @@
 import cProfile
-import spyql.cli
```

Comment:
Removed unused import

Reply:
This brakes the code if you try to run in "profiler" mode:

```
python3 -m spyql.prof "SELECT 1"
```

because of:

https://github.com/dcmoura/spyql/blob/8445350cef8c336978274fb9de4ad0d2359f2091/spyql/prof.py#L10

flake8 can't see the code being executed in the string.

So I suggest keeping the import and adding an inline ignore flag: 

```
import spyql.cli # noqa: F401
```


This also shows that we should add a test case that tests calling the profile, but we can do that in another PR.

Reply:
Oh... I fixed it.

Reply:
OK, now we are good :-)